doc/manual.asciidoc

```diff
@@ -917,6 +917,8 @@ paths, where a space would otherwise separate filenames.  See below.)
 `$:` :: a colon.  (This is only necessary in `build` lines, where a colon
 would otherwise terminate the list of outputs.)
 
+`$^` :: a newline. Inserts '\n' into the resulting string.
```

Comment:
Why it named not a `$n`, similar to the `$:`?

Reply:
Not so visible when surrounded with an another text, and ^ usually means the beginning like in regular expressions.